src/vm/i386/jithelp.S

```diff
@@ -399,23 +399,23 @@ UniversalWriteBarrierHelper CheckedWriteBarrier
 UniversalWriteBarrierHelper WriteBarrier
 #endif // FEATURE_USE_ASM_GC_WRITE_BARRIERS
 
-WriteBarrierHelper EAX
-WriteBarrierHelper EBX
-WriteBarrierHelper ECX
-WriteBarrierHelper ESI
-WriteBarrierHelper EDI
-WriteBarrierHelper EBP
+CheckedWriteBarrierHelper EAX
+CheckedWriteBarrierHelper EBX
+CheckedWriteBarrierHelper ECX
+CheckedWriteBarrierHelper ESI
+CheckedWriteBarrierHelper EDI
+CheckedWriteBarrierHelper EBP
 
 ByRefWriteBarrierHelper
 
-LEAF_ENTRY JIT_WriteBarrierLast, _TEXT
+LEAF_ENTRY JIT_CheckedWriteBarrierLast, _TEXT
     ret
-LEAF_END JIT_WriteBarrierLast, _TEXT
+LEAF_END JIT_CheckedWriteBarrierLast, _TEXT
 
 // This is the first function outside the "keep together range". Used by BBT scripts.
-LEAF_ENTRY JIT_WriteBarrierEnd, _TEXT
+LEAF_ENTRY JIT_CheckedWriteBarrier_End, _TEXT
     ret
-LEAF_END JIT_WriteBarrierEnd, _TEXT
+LEAF_END JIT_CheckedWriteBarrier_End, _TEXT
 
```

Comment:
It would be better to use ``LEAF_END_MARKED`` instead of explicitly appending ``_End``.